portfolio/src/main/webapp/script.js

```diff
@@ -114,9 +117,24 @@ function postMarker(lat, lng, type, date, time, address, description) {
   fetch('/markers', {
     method: 'POST',
     body: params
-  }).catch((error) => {
+  }).then(response => response.json())
+  .then(result => { 
+    
+    if (result.status != 'SUCCESS') {
+      displayRepeatedMarkerUI();
+      
+      var textError = (result.failure == 'REPEAT') ? "Repeated crimes" : "Unknown failure";
```

Comment:
You want more descriptive messages, which tells a user what they should do next.

Reply:
Ok, will do that!